File: src/main/java/me/kuku/yuq/service/NeTeaseService.java
package me.kuku.yuq.service;

import com.IceCreamQAQ.Yu.annotation.AutoBind;
import me.kuku.yuq.entity.NeTeaseEntity;

import java.util.List;

@AutoBind
public interface NeTeaseService {
    NeTeaseEntity findByQQ(Long qq);
    void save(NeTeaseEntity neTeaseEntity);
    List<NeTeaseEntity> findAll();
    int deByQQ(Long qq);
}

File: src/main/kotlin/me/kuku/yuq/service/NeTeaseService.kt
package me.kuku.yuq.service

import com.IceCreamQAQ.Yu.annotation.AutoBind
import me.kuku.yuq.entity.NeTeaseEntity

@AutoBind
interface NeTeaseService {

    fun findByQQ(qq: Long): NeTeaseEntity?
    fun save(neTeaseEntity: NeTeaseEntity)
    fun findAll(): List<NeTeaseEntity>
    fun delByQQ(qq: Long): Int

}